### PR TITLE
Search hookup louiew

### PR DESCRIPTION
**Pull request by dustyfingers**
## PR TITLE

**Description:** 
wired up basic search functionality on front end

  
  
  
  

### Checklist

 * [ ] My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my own code
 * [ ] My code has been reviewed by at least one peer
 * [ ] I have commented my code, particularly in hard-to-understand areas
 * [ ] I have made corresponding changes to the documentation
 * [ ] My changes generate no new warnings
 * [ ] I have added tests that prove my fix is effective or that my feature works
 * [ ] New and existing unit tests pass locally with my changes
 * [ ] There are no merge conflicts

### CODE REVIEW

File: src/assets/SearchResultCard.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import moment from 'moment';
+import { List, Space, Divider, Avatar } from 'antd';
+import { useRouteMatch, Link } from 'react-router-dom';
+
+const SearchResultCard = ({ cardType, content }) => {
+    const { url } = useRouteMatch();
+    (cardType === "post") && console.log({content});
```
Comment (Plofland) [inline src/assets/SearchResultCard.js:8]: We can get rid of this console.log if you don't need it.

File: src/index.js [hunk 2]
```diff
@@ -8,7 +9,7 @@ import { BrowserRouter as Router } from 'react-router-dom';
 import App from './App';
 import './index.css';
 
-const store = createStore(reducer, applyMiddleware(thunk));
+const store = createStore(reducer, applyMiddleware(thunk, logger));
```
Comment (Plofland) [inline src/index.js:12]: Nice, I can really use this logger for testing.

File: src/store/actions/index.js [hunk 3]
```diff
@@ -332,6 +332,7 @@ export const retrieveFullSearchResults = (search) => (dispatch) => {
   axiosWithAuth()
     .get(`${BACKEND_URL}/api/search/full/${search}`)
     .then((res) => {
+      console.log(res);
```
Comment (Plofland) [inline src/store/actions/index.js:335]: console.log here

**Review by Plofland — APPROVED** [review]
Nice job mate, can't wait to see it up and running after this merge.

**Review by ialkamal — APPROVED** [review]
I'll merge this and checkout the search functionality on the FE.

**Pull request merged**